pkg/urbit/vere/term.c

```diff
@@ -919,7 +919,7 @@ u3_term_get_blew(c3_l tid_l)
   c3_l           col_l, row_l;
 
   struct winsize siz_u;
-  if ( uty_u && (0 == ioctl(uty_u->fid_i, TIOCGWINSZ, &siz_u)) ) {
+  if ( c3y == u3_Host.ops_u.tem && uty_u && (0 == ioctl(uty_u->fid_i, TIOCGWINSZ, &siz_u)) ) {
```

Comment:
Please split this across multiple lines in the (inconsistently enforced) house style:

```
if ( (foo) &&
      (bar) &&
      (baz) )
{
  // stuff
}
```